.github/workflows/makefile-test.yaml

```diff
@@ -19,29 +19,31 @@ jobs:
         uses: actions/setup-python@v3
         with:
           python-version: ${{ matrix.python-version }}
-      - name: Install dependencies
+      - name: Install latest CBMC
+        id: install_latest_cbmc
+        shell: bash
         run: |
-          get_latest_release() {
-            curl --silent "https://api.github.com/repos/$1/releases/latest" |
-            jq -r '.tag_name'
-          }
-          cbmc_latest_release=$(get_latest_release diffblue/cbmc)
-          litani_latest_release=$(get_latest_release awslabs/aws-build-accumulator)
-
-          curl -o cbmc.deb -L \
-            https://github.com/diffblue/cbmc/releases/download/$cbmc_latest_release/ubuntu-20.04-$cbmc_latest_release-Linux.deb
-          curl -o litani.deb -L \
-            https://github.com/awslabs/aws-build-accumulator/releases/download/$litani_latest_release/litani-$litani_latest_release.deb
-
-          sudo apt-get update \
-            && sudo apt-get install --no-install-recommends --yes \
-                                            ./cbmc.deb \
-                                            ./litani.deb \
-                                            scdoc \
-                                            mandoc
-
-          python3 -m pip install pyyaml jinja2 cbmc-viewer
-
+          # Search within 5 most recent releases for latest available package
+          CBMC_REL="https://api.github.com/repos/diffblue/cbmc/releases?page=1&per_page=5"
+          CBMC_DEB=$(curl -s $CBMC_REL | jq -r '.[].assets[].browser_download_url' | grep -e 'ubuntu-20.04' | head -n 1)
```

Comment:
Suggest `curl -o` to save to a file so that the script fails early if this doesn't work. Then define the variable separately. I also recommend to check that the variable is non-empty, because you're `curl`ing it 2 lines down so it's good to fail early if the URL doesn't exist for any reason.

Reply:
If a failure occurs when trying to determine `CBMC_DEB`, then that variable will be empty.
As a result, the workflow will fail when `CBMC_ARTIFACT_NAME` is created. A failure here means that fetching an Ubuntu 20.04 download URI within the latest 5 releases didn't work. Therefore, an "early exit" does take place as the second `curl` 2 lines down, as you mentioned, won't even be executed.